my-app/src/css/dropDown.css

```diff
@@ -0,0 +1,15 @@
+#dropDownMenuLeft{
```

Comment:
it's better to be using class selectors `.dropDownMenu` instead of id selectors `#dropDownMenuLeft`.

All your drop down menus should look alike and one aspect of css is it can save you a lot of duplicated code. You should write css styles for all your drop down menus at once, in one location. Instead of writing css styles for every individual component, which is unmaintainable.

Check if you have duplicate css in places and if you can consolidate them into class selectors.

For example, the "Submit" button on the "Create Your own File" page can have the same style as the "Home" button on the file page.